46.md

```diff
@@ -96,4 +117,10 @@ The signer key will always be the key of the user who controls the signer device
 - **ping**
   - params: []
   - result: `"pong"`
+- **create_account**
```

Comment:
when do you call `create_account`? before calling `connect`?

Reply:
> it will reply with a new pubkey <pubkey1>

How does this reply come through? Eg what type of event are we subscribing to for connect calls?

Reply:
just the standard nip-46 protocol, a `kind:24133` p-tagging the key you are writing from (in this case the client/local key).